apps/server/src/modules/learnroom/mapper/copy.mapper.ts

```diff
@@ -2,7 +2,7 @@ import { CopyStatus, CopyStatusEnum, EntityId, Lesson, Task } from '@shared/doma
 import { LessonCopyApiParams } from '@src/modules/learnroom/controller/dto/lesson/lesson-copy.params';
 import { LessonCopyParentParams } from '@src/modules/learnroom/uc/lesson-copy.uc';
 import { TaskCopyApiParams } from '@src/modules/task/controller/dto/task-copy.params';
-import { TaskCopyParentParams } from '@src/modules/task/uc/task-copy.uc';
+import { TaskCopyParentParams } from '@src/modules/learnroom/uc/task-copy.uc';
```

Comment:
Task copy service should be a service not the UC and it should located in tasks module.

Reply:
Final organisation of files in modules is planed here: [BC-2417](https://ticketsystem.dbildungscloud.de/browse/BC-2417)